_posts/2023-11-30-indico-3-3-news.md

```diff
@@ -0,0 +1,96 @@
+---
+layout:     post
+title:      "Indico 3.3"
+date:       2023-11-30 12:00
+author:     "Indico Team"
+categories: indico release
+---
+
+Today, we are introducing Indico 3.3! This upcoming release brings a lot of new features and improvements, including a **new Check-in app**, **accessibility improvements (a11y)**, **receipt generation** and a whole lot more.
+
+> note ""
+> The full change log can be found [here](https://docs.getindico.io/en/stable/changelog/#v3-3).
+
+---
+
+## ♿️ Accessibility improvements (a11y)
+
+The accessibility of Indico has been improved in this release, in-line with the [WCAG 2.1](https://www.w3.org/TR/WCAG21/). This means that Indico is now easier to use by people with disabilities, including those who are blind, deaf, or have limited mobility. Notable improvements include:
+
```

Comment:
We may want to double check this part - do we have any a11y improvements for people who are deaf or have limited mobility?

Reply:
We'll update the blogpost with everything that we manage to merge before the public release :)